### DISCUSSION TITLE
REST API responds wrong data

### DISCUSSION THREAD

**Discussion by juaninicolai** [microcks/microcks#1244] [Q&A] [answered]
Hello, I have uploaded a REST API using a Postman sample (including HTTP request and saved API response).
I can see on the Microcks UI that everything is correct, and the intended response data is complete.
But when I send a request to the microcks generated endpoint, the response data comes wrong.

This is the Postman collection with the response data [ContainerDetails.postman_collection.json](https://github.com/user-attachments/files/16310796/ContainerDetails.postman_collection.json)

And in the picture attached you can see the response that I am getting (only the first element of the list, when it should return the whole list of objects)

<img width="886" alt="Screenshot 2024-07-19 at 12 33 00" src="https://github.com/user-attachments/assets/4e56338b-c958-4d55-98c7-a2a9ee5fd688">

Has anyone experienced the same or knows of a workaround this?

Thanks in advance, Juan

**Comment by juaninicolai** [comment, marked as answer]
Found the answer on this [discussion](https://github.com/orgs/microcks/discussions/682)

**Reply by lbroudoux** [reply]
Hey! Here's the doc reference that may be of interest: https://microcks.io/documentation/references/artifacts/openapi-conventions/
